blueocean-dashboard/package.json

```diff
@@ -14,8 +14,8 @@
     "test:watch": "gulp test:debug",
     "bundle": "gulp bundle",
     "bundle:watch": "gulp bundle:watch",
-    "mvnbuild": "gulp bundle",
-    "mvntest": "gulp test lint"
+    "mvnbuild": "gulp lint bundle",
```

Comment:
why lint in build and not in build?

Further did I understood wrong that this PR had been to drop redux? If so why it is still a dep?